backend/src/test/java/gov/cdc/usds/simplereport/api/BaseApiTest.java

```diff
@@ -22,16 +24,29 @@
 import com.graphql.spring.boot.test.GraphQLResponse;
 import com.graphql.spring.boot.test.GraphQLTestTemplate;
 
+import gov.cdc.usds.simplereport.config.AuthorizationProperties;
 import gov.cdc.usds.simplereport.config.authorization.OrganizationRole;
 import gov.cdc.usds.simplereport.config.authorization.OrganizationRoles;
 import gov.cdc.usds.simplereport.service.AuthorizationService;
 import gov.cdc.usds.simplereport.service.OrganizationInitializingService;
 import gov.cdc.usds.simplereport.service.model.IdentitySupplier;
-import gov.cdc.usds.simplereport.service.OktaService;
 import gov.cdc.usds.simplereport.test_util.DbTruncator;
 import gov.cdc.usds.simplereport.test_util.TestUserIdentities;
 
-@SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT)
+import com.okta.spring.boot.sdk.config.OktaClientProperties;
+import com.okta.sdk.authc.credentials.TokenClientCredentials;
+import com.okta.sdk.client.Client;
+import com.okta.sdk.client.Clients;
+import com.okta.sdk.resource.user.User;
+import com.okta.sdk.resource.group.Group;
+
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+
+@SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT,
+                properties = {
+                    "simple-report.authorization.role-prefix=TEST-TENANT:",
+})
+@EnableConfigurationProperties({OktaClientProperties.class, AuthorizationProperties.class})
```

Comment:
The EnableConfigurationProperties annotation should not be needed—this is a full context. Also I don't think we should change the role prefix here.